packages/Webkul/Core/src/Helpers/Exchange/ExchangeRates.php

```diff
@@ -53,7 +53,15 @@ public function updateRates()
                 continue;
             }
 
-            $result = $client->request('GET', $this->apiEndPoint . '?access_key='. $this->apiKey . '&base=' . config('app.currency') . '&symbols=' . $currency->code);
+            $result = $client->request(
+                'GET',
+                $this->apiEndPoint . '?to=' . $currency->code . '&from=' . config('app.currency') . '&amount=1', [
```

Comment:
I think there must be an argument from where you can add params. Just check that and remove the `?` dependency.

Reply:
Done